docs/02.API-REFERENCE.md

```diff
@@ -1486,6 +1486,48 @@ jerry_value_set_error_flag (jerry_value_t *value_p);
 - [jerry_value_t](#jerry_value_t)
 
 
+## jerry_get_value_without_error_flag
+
+**Summary**
+
+If the input value is a error value, then return its reference value.
+Otherwise return a new reference to itself.
```

Comment:
If the input value is an error value, then return a new reference to its referenced value.
Otherwise, return a new reference to the value itself.

also update the comment for the function.